### DISCUSSION TITLE
v2.2.3

### DISCUSSION THREAD

**Discussion by EskiMojo14** [reduxjs/redux-toolkit#4316] [Announcements]
This minor release fixes the types for functions that accept a React Context instance to match the [changes in React Redux v9](https://redux-toolkit.js.org/usage/migrating-rtk-2#custom-context-typing).

## What's Changed
* Update React Redux dependency to v9, and update docs to use `.withTypes` by @aryaemami59 in https://github.com/reduxjs/redux-toolkit/pull/4308

**Full Changelog**: https://github.com/reduxjs/redux-toolkit/compare/v2.2.2...v2.2.3

<hr /><em>This discussion was created from the release <a href='https://github.com/reduxjs/redux-toolkit/releases/tag/v2.2.3'>v2.2.3</a>.</em>

**Comment by Radiant-F** [comment]
that was fast.

**Reply by EskiMojo14** [reply]
it was a bug fix that i wanted to make sure was out - because frankly, it shouldn't have been missed for 2.0 😛
